internal/test/helpers/mod.go

```diff
@@ -41,6 +43,7 @@ func NewMod(path string) (Mod, error) {
 	}, nil
 }
 
+// FindDependencyVersion return the version of a go.mod dependency.
 func (m Mod) FindDependencyVersion(dependency string) (string, error) {
```

Comment:
Do we even use this?

Found nothing, maybe in your vcsim branch?

Reply:
I just moved stuff around, we can clean up as a follow up